src/app/shapes/Ellipse.ts

```diff
@@ -0,0 +1,35 @@
+import * as Konva from "konva";
+import {Shape} from "./Shape";
+
+export class Ellipse extends Shape{
+
+  public instance: Konva.Ellipse;
```

Comment:
instance is on shape now so remove this here

Reply:
We need to define it so that we can use Ellipse specific functionality.